docs/channels/emails.rst

```diff
@@ -225,15 +225,53 @@ Marketers can place the signature into an Email using the ``{signature}`` token.
 Email delivery
 ##############
 
-Mautic delivers Emails using the method defined by the system administrator. If you are the system administrator for your Company, then you need to add the Email protocol for your Mautic instance to use. Mautic integrates with any Email service provider which offers SMTP mail servers as well as several distinct services such as:
+Mautic delivers Emails using the method defined by the system administrator. If you are the system administrator for your Company, then you need to add the Email protocol for your Mautic instance to use. Mautic integrates with any Email service provider which offers SMTP mail servers. As Mautic uses the :xref:`Symfony Mailer` library since v5, it supports its core plugins. Transports for other email services might be found on Github or Packagist.
 
-- :xref:`Mandrill`
-- :xref:`GMail`
-- :xref:`Sendgrid`
-- :xref:`Mailjet`
-- :xref:`Postmark`
-- :xref:`Sendmail`
-- :xref:`Amazon SES`.
+SMTP Transport
+**************
+
+The SMTP transport is the default transport for Mautic. It's configured in the Mautic configuration under the Email Settings tab. The configuration is the same as in the :xref:`Symfony Mailer` documentation.
+
+This is the example DSN mentioned in the :xref:`Symfony Mailer` documentation for SMTP:
+
+.. code-block:: shell
+    
+    smtp://user:pass@smtp.example.com:port
+
+This is how this DSN converted to Mautic's email configuration:
```

Comment:
I think this is what you mean?

```suggestion
Mautic creates this automatically from the values entered in the Email configuration:
```

@escopecz please add a table which explains what each of the values actually means. Should just need three columns I guess or maybe two - value name, what it means, any notes / links / references.  Docs on how to do the markdown are here: https://sphinx-rtd-theme.readthedocs.io/en/stable/demo/lists_tables.html#tables. Thanks!

Reply:
Will do

Reply:
The table was added in https://github.com/mautic/user-documentation/pull/174/commits/7411e6172d793b1b2b1345ddfaf6192b8248b2e5